nidata/multimodal/hcp/__init__.py

```diff
@@ -226,6 +260,8 @@ def fetch(self, n_subjects=1, data_types=None, volume_types=None,
             data_types = ['anat', 'diff', 'func', 'rest']
         if volume_types is None:
             volume_types = ['3T']  # fsaverage_LR32k, Native
```

Comment:
I can't remember clearly (I've been very sick, sorry), but did we decide to keep `volume_types`? I thought not... but my memory is foggy (and energy level very low to check)